### PR TITLE
Link to static html list from no-js raf tool

### PR DESCRIPTION
**Pull request by wpears**
Alternative to https://github.com/cfpb/consumerfinance.gov/pull/6994

Instead of shipping a bunch of html to every user, think just gives `no-js` users a *link* to said html. Here's a link to that html: https://files.consumerfinance.gov/data/rental-assistance.html

I think this is a worse experience for `no-js` users than https://github.com/cfpb/consumerfinance.gov/pull/6994, but definitely a better experience for the 99% other users, so probably better overall.

Testing:
 - Pull branch
 - Visit http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/
 - Things should work as normal
 - Open devtools and use cmd + shift +p to open the run panel, type Disable Javascript and run that command when it populates
 - Refresh the page
 - The page should render with the following notification:
 
Screen Shot 2022-04-05 at 4 37 54 PM

That link should point to https://files.consumerfinance.gov/data/rental-assistance.html

### CODE REVIEW

**Review by contolini — APPROVED** [review]
I dig this over #6994 for the performance reasons we discussed yesterday. Thanks for doing this!

File: cfgov/jinja2/v1/_includes/blocks/raf_tool.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
     {{ notification.render(
       'error',
       true,
-      'This tool is not supported in your browser.
-       Please try a newer browser or confirm that JavaScript is enabled.'
+      'This interactive tool is not supported in your browser.
+       <a style="text-decoration:underline" href="https://files.consumerfinance.gov/data/rental-assistance.html">Please view the data directly</a>.'
```
[the remark this answers is not in the document]
Comment (marteki) [reply cfgov/jinja2/v1/_includes/blocks/raf_tool.html:29]: @contolini The language change is more descriptive, but is less clear and less plain language. Is there a simpler word that we can use besides "incompatible"?

[the remark this answers is not in the document]
Comment (wpears) [reply cfgov/jinja2/v1/_includes/blocks/raf_tool.html:29]: Screen Shot 2022-04-06 at 12 06 38 PM

How about this?

[the remark this answers is not in the document]
Comment (contolini) [reply cfgov/jinja2/v1/_includes/blocks/raf_tool.html]: > Screen Shot 2022-04-06 at 12 06 38 PM
> 
> How about this?

Love it.

**Comment by marteki** [comment]
As mentioned on #6994: let's use this PR to address the problem, and iterate on it soon to meet the CFPB Design System guidelines on appearance.

**Review by virginiacc — APPROVED** [review]
I concur with the general preference for this approach. Thanks for making this improvement, @wpears!

**Pull request merged**